### ISSUE TITLE
init with existing .pingy.json

### ISSUE THREAD

**Issue by borekb** [pingyhq/pingy-cli#29]
I went through `pingy init` and successfully bootstrapped my project. Then I tried removing `node_modules` as they won't be committed into Git anyway. I ran `pingy init` again and my expectation was that it would just run `npm install` and output some message. However, it started asking all the questions again. How is this supposed to work?

BTW, just discovered Pingy CLI, it's an amazing tool. Thanks for it!

**Comment by davej** [comment]
Yeah currently it is just for initialisation of the project, and not for changing an existing project. This may change in the future but right now, I'll keep it simple to give me time to focus on other features.

Thanks for the bug report.

**Comment by borekb** [comment]
Maybe if `init` detects existing `.pingy.json`, it could just print out a message like:

```
👍 Pingy was already initialized, just run `npm install & pingy dev`
```

and finish.

**Comment by davej** [comment]
Good idea. 👍

**Comment by davej** [comment]
Fixed in [0.6.9](https://github.com/pingyhq/pingy-cli/releases/tag/%40pingy%2Fcli%400.6.9)

Pingy now detects existing `.pingy.json` and `package.json` and asks users to confirm if they wish to continue

![image](https://user-images.githubusercontent.com/26971/27126572-d0b47644-50ef-11e7-9a71-4d8fac78698f.png)
